### DISCUSSION TITLE
Security issue reported on version 3.57.3

### DISCUSSION THREAD

**Discussion by smknstd** [spatie/browsershot#691] [Bugs]
https://github.com/advisories/GHSA-6q49-35h6-rq2p

Hello ! I don't see any fix or PR related to this. Did I miss something ?

**Comment by freekmurze** [comment]
This has been fixed in 3.57.4, and final details will be fixed by https://github.com/spatie/browsershot/pull/682

**Comment by smknstd** [comment]
Ok, it works for me

![Screenshot from 2022-12-05 09-29-40](https://user-images.githubusercontent.com/2412608/205590289-ce040231-542f-4820-9da6-89512b55bfb6.png)

I was confused because the fix is missing from the release note https://github.com/spatie/browsershot/releases/tag/3.57.4
